requirements.txt

```diff
@@ -1,12 +1,12 @@
-# Setup
+# setup
 colorama>=0.4.3
 crunch64>=0.3.1,<1.0.0
 ipl3checksum>=1.2.0,<2.0.0
 
 # disasm
 rabbitizer>=1.0.0,<2.0.0
```

Comment:
This is just a comment in general (and perhaps solicits a wider discussion...), can we adopt pinning exact versions? Loose version constraints are okay, but pinning exact versions would offer more control over the environment (venvs in this case) and prevent unexpected changes due to package updates. It would also facilitate troubleshooting.

Reply:
I'm not a huge fan of pinning exact versions. Or at least not for the libraries that we have control over.
We usually add new features and fix issues pretty quickly, it would be annoying to be out of date and need to do PRs that only do version bumps on the requirements.txt file

Reply:
Yeah, our packages already adhere to semver so it shouldn't be an issue